x-pack/plugin/core/template-resources/src/main/resources/entsearch/connector/chat/support-template.json

```diff
@@ -0,0 +1,67 @@
+{
+  "index_patterns": ["content-chat-support-v1-*"],
+  "template": {
+    "mappings": {
+      "properties": {
+        "metadata": {
+          "properties": {
+            "support": {
+              "properties": {
+                "status": {
+                  "type": "keyword"
+                },
+                "assignee": {
+                  "properties": {
+                    "name": {
+                      "type": "text",
+                      "fields": {
+                        "keyword": {
+                          "type": "keyword"
+                        }
+                      }
+                    },
+                    "email_address": {
+                      "type": "keyword"
+                    },
+                    "id": {
+                      "type": "keyword"
+                    }
+                  }
+                },
+                "category": {
+                  "type": "keyword"
+                },
+                "customer": {
```

Comment:
Nit; rename to `contact` ? 

In salesforce we have account that can have multiple contacts and a contact can open a salesforce case. Here it's unclear if you are referencing a customer account (org) or a contact

We might need to add `accountId` at least to reference parent organisation

Reply:
Dana was also thinking that just `account` may be better